test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -285,6 +322,20 @@ public void wipeCluster() throws Exception {
         adminExecutionContext.callApi("snapshot.delete_repository", deleteSnapshotsArgs, Collections.emptyList(), Collections.emptyMap());
     }
 
+    /**
+     * @return An exclude set of indices that will not be removed in between tests.
+     */
+    protected Set<String> excludeIndices() {
```

Comment:
knowing how the exclusions stuff works in es, and the fact that it depend on the order of the elements, I think using a set is not the best here. WDYT?